### PR TITLE
About us page design and content updated

### LINKED ISSUE #8: About us page design and content
- [ ] content
- [ ] designing

For the content you can refer to the old [website](https://codeiiest.github.io/#welcome).

### PR DESCRIPTION
**Pull request by REX51**
This fixes #8

### CODE REVIEW

File: pages/about.vue [hunk 1]
```diff
@@ -18,15 +19,31 @@
...
 .js{
-    text-align: justify;
-    text-justify: auto;
+    text-align: center;
+    font-size: 1.2rem;
+    line-height: 2;
+    max-width: 38em;
+    margin: auto;
+    /* color: #4a4a4a;
+    background-color: #f9f9f9; */
```
Comment (raydwaipayan) [inline pages/about.vue:40]: Please remove commented code

File: pages/about.vue [hunk 2]
```diff
@@ -18,15 +19,31 @@
...
+    padding: 13px;
+}
+.headtext{
+    color: red;
+    font-size: 1.2rem;
+    border-bottom: 2px solid currentColor;
 }
+
 </style>
```
Comment (raydwaipayan) [inline pages/about.vue:49]: newline at end of file please.

**Review by raydwaipayan — CHANGES_REQUESTED** [review]
Few minor changes. Otherwise looks good @REX51

File: pages/about.vue (hunk 1 above)
Comment (REX51) [reply pages/about.vue:40]: Okay sure

**Review by raydwaipayan — APPROVED** [review]
Looks Good @REX51 . Thanks for contributing!

**Pull request merged**